src/ports/server/component.ts

```diff
@@ -668,6 +697,91 @@ export async function createServerComponent({
       return res.sendStatus(200)
     })
 
+    // Generate identity endpoint
+    app.post('/identities', async (req: Request, res: Response) => {
+      try {
+        // Extract AuthIdentity from request body
+        const { identity }: { identity: AuthIdentity } = req.body
+
+        if (!identity) {
+          return sendResponse<InvalidResponseMessage>(res, 400, {
+            error: 'AuthIdentity is required in request body'
+          })
+        }
+
+        // Validate auth chain using the same logic as /requests endpoint
+        try {
+          await validateAuthChain(identity.authChain)
```

Comment:
You need to validate here that the ephemeral wallet address that is being sent, as part of the identity, is the `finalAuthority` of the AuthChain in order to be sure that the ephemeral wallet is the same being authorized by the auth chain. Would you mind adding that check?